programs/temp/src/state.rs

```diff
@@ -60,6 +60,19 @@ impl OffsetTiers {
         self.authority = input.authority;
         self.levels = input.levels;
     }
+
```

Comment:
Line 15 (GlobalStateInput). 

Do we need the bump in the input struct? I think you can get this from ctx.bumps

See [here](https://github.com/sunrise-stake/app/blob/develop/programs/sunrise-stake/src/instructions/init_lock_account.rs) for an example.

Having it in the GlobalState account itself is correct though.